### PR TITLE
Nitrium rework

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Nitrium now adds an organ that allows you to "overclock" your adrenal glands to gain antistun and speed

Very similar to changeling adrenals

Applies jitter when removing stamina damage and also has glowing red veins like a shadowling, so security can tell you apart from a changeling

You need to have the high metabolism to get speed boost otherwise it's just antistun

# Wiki Documentation

Still removes 10 stamina per tick passively

10 second cooldown

applies 10 burn and 5 toxin damage and 35 disgust stacks on use (will vomit if used twice) 

removes stuns and removes 75 stam on use

cannot be used while unconscious

adds 10 units of /datum/reagent/medicine/changelingadrenaline (called nitrium adrenaline now) and 4 units of /datum/reagent/medicine/nitriumhaste

nitrium haste is identical to changeling haste but has -0.55 slowdown instead of -2.0

# Changelog

:cl:  
tweak: Nitrium has been reworked into an active effect rather than a passive one
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/nitrium.dm [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	name = "adrenaline gland"
+	desc = "An adrenal gland, this one is incredibly swollen."
+	zone = BODY_ZONE_CHEST
+	slot = ORGAN_SLOT_ADRENAL
+	icon_state = "blacktumor"
+	var/datum/action/cooldown/adrenal_tumor/implant_ability
+	var/mutable_appearance/ad_overlay
+	var/upgraded = FALSE
+
+/obj/item/organ/nitrium_tumor/Insert(mob/living/carbon/M)
```
Comment (ThatLing) [inline code/modules/surgery/organs/nitrium.dm]:
```suggestion
/obj/item/organ/nitrium_tumor/Insert(mob/living/carbon/M, special = 0, drop_if_replaced = TRUE)
```

File: code/modules/surgery/organs/nitrium.dm [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	. = ..()
+	implant_ability = new
+	implant_ability.tumor = src
+	implant_ability.Grant(M)
+	// red glowing veins as a tell for sec, surely this is healthy for you
+	ad_overlay = mutable_appearance('yogstation/icons/mob/clothing/suit/suit.dmi', "sl_shell", -UNDER_SUIT_LAYER)
+	M.add_overlay(ad_overlay)
+
+/obj/item/organ/nitrium_tumor/Remove(mob/living/carbon/M)
```
Comment (ThatLing) [inline code/modules/surgery/organs/nitrium.dm]:
```suggestion
/obj/item/organ/nitrium_tumor/Remove(mob/living/carbon/M, special = 0)
```

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: code/modules/surgery/organs/nitrium.dm [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+	name = "adrenaline gland"
+	desc = "An adrenal gland, this one is incredibly swollen."
+	zone = BODY_ZONE_CHEST
+	slot = ORGAN_SLOT_ADRENAL
+	icon_state = "blacktumor"
+	var/datum/action/cooldown/adrenal_tumor/implant_ability
+	var/mutable_appearance/ad_overlay
+	var/upgraded = FALSE
+
+/obj/item/organ/nitrium_tumor/Insert(mob/living/carbon/M, special = 0, drop_if_replaced = TRUE)
```
Comment (ThatLing) [inline code/modules/surgery/organs/nitrium.dm]:
```suggestion
/obj/item/organ/nitrium_tumor/Insert(mob/living/carbon/M, special = FALSE, drop_if_replaced = FALSE)
```
oh wait you prob dont wanna drop this

**Comment by Moltijoe** [comment]
https://github.com/yogstation13/Yogstation/pull/18591 was approved instead

**Pull request closed without merging**